test/parallel/test-repl-persistent-history.js

```diff
@@ -247,10 +247,7 @@ function runTest(assertCleaned) {
     useColors: false,
     terminal: true
   }, function(err, repl) {
-    if (err) {
-      console.error(`Failed test # ${numtests - tests.length}`);
```

Comment:
It's a pity that `assert.ifError()` doesn't take an optional message argument, unlike every other assert method...

![image](https://cloud.githubusercontent.com/assets/15943089/21568282/e5c9d940-cebb-11e6-9336-633416586359.png)